code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -339,6 +352,10 @@
 	piping_layer = 4
 	icon_state = "scrub_map_on-4"
 
+/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer5
+	piping_layer = 5
+	icon_state = "scrub_map_on-4"
+
```

Comment:
These seem quite questionable because there is no relevant icon states. Check image

![image](https://github.com/user-attachments/assets/2b8d495f-4403-4524-9136-241684181f13)